src/app/molecules/message/TimelineChange.jsx

```diff
@@ -19,22 +20,25 @@ function TimelineChange({
 
   switch (variant) {
     case 'join':
-      iconSrc = JoinArraowIC;
+      iconSrc = JoinArrowIC;
       break;
     case 'leave':
-      iconSrc = LeaveArraowIC;
+      iconSrc = LeaveArrowIC;
       break;
     case 'invite':
-      iconSrc = InviteArraowIC;
+      iconSrc = InviteArrowIC;
       break;
     case 'invite-cancel':
-      iconSrc = InviteCancelArraowIC;
+      iconSrc = InviteCancelArrowIC;
       break;
     case 'avatar':
       iconSrc = UserIC;
       break;
+    case 'edit':
+      iconSrc = HashIC;
+      break;
```

Comment:
```suggestion
    case 'room-profile':
      iconSrc = HashIC;
      break;
```
`variant` propTypes for this is also missing.